Makefile

```
@@ -325,243 +209,355 @@ clean-cfg:
-	RUSTFLAGS="${RUSTFLAGS}" RUST_BACKTRACE=1 RUST_TEST_THREADS=1 CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUNNER='sudo -E' cargo test real_ -- --skip clevis_real_
+.PHONY: clippy-utils
+## Run clippy on the utils binary
+clippy-utils:
+	cargo ${CLIPPY} ${CLIPPY_OPTS} ${UTILS_FEATURES}
 
-## Basic tests
-test:
-	RUSTFLAGS="${RUSTFLAGS}" RUST_BACKTRACE=1 cargo test --all-features -- --skip real_ --skip loop_ --skip clevis_ --skip test_stratis_min_ --skip test_stratisd_min_
+.PHONY: docs-cli
+## Build docs-rust for CI
+docs-ci: docs-rust
```

checkmake (0.3.2)

[warning] Target "docs-ci" should be declared PHONY.

(phonydeclared)